src/js/components/useToast.test.ts

```diff
@@ -37,92 +37,94 @@ describe('useToast', () => {
 
     it('should display the HTML markup message when used', () => {
       const toast = useToast(Toastify.TestMarkupMessage);
-      toast.instance.show();
```

Comment:
```
const {instance: toast} = useToast(Toastify.TestMarlupMessage);
toast.show()
```

Is it that hard to understand destructuring?

Reply:
In above code, you used bootstrap show(), if the useToast not initialized correctly (invalid template or invalid markup) it will stop JS execution.

Reply:
Bootstrap doesn't control toast instance before  show() or hide();

Reply:
Inside React/Vue composables, the good practice is to have an error that you can get while destructuring which is undefined if there are no errors

```
const {instance: toast, element, error} = useToast(Toastify.TestMarlupMessage);

if (!error) {
  toast.show()
  element.remove()
}
```

where `error` would be something like `'The toast markup override is invalid.'`

Reply:
You can also verify that the element exist

```
if (!toast) return;
```

or 
```
toast?.show();
```

Reply:
You can not get error after initializing, Bootstrap throws error in runtime.
`new Toast('.dummy');`
Instance will be created without error by BS Toast, but ".dummy" is not exists in the DOM.
So using `show()` throws an error.

Reply:
Ok, just tested

Can we at least make these override methods (show, hide, dispose and more) inside a `toast` property that the `useToast` object return ?

So you will be able to use 

`const {element, toast, content} = useToast(...)` where toast === `{show, hide, dispose, instance}` and instance === Bootstrap.Toast.instance ?